### DISCUSSION TITLE
My domain

### DISCUSSION THREAD

**Discussion by ShailendraMadarie** [serverless-dns/serverless-dns#167] [Q&A]
I have set up my owns domain in cloudflare workers for the serverless dns.
However when i go to my domain i set it up like dns.mydomain.com it redirects me to rethinkdns.com website..als the doh urls are rethinkdns.com 's
Is there something more i should do make it work with my own domain?

**Comment by ignoramous** [comment]
That's expected if you're visiting `dns.mydomain.com` from your browser. The webpage that loads is where you can set blocklist rules and obtain a URL that conforms to those rules.

https://flyto.rethinkdns.com/ is our test domain, and if you visit that link from your browser (the test domain is currently down), the resulting page on Rethink's website would let you set blocklists and generate a DoH URL for `flyto.rethinkdns.com`, like so: https://flyto.rethinkdns.com/1:OPD_____9_8A_AYA4dMAoA==

**Reply by suricloud** [reply]
But if we self host on cloudflare,
Can you update the giithub script that we can use our own domain like https://rdns.domain.com/config?
Now if use our domain you get redirected to rethinkdns.com/config.

**Reply by ignoramous** [reply]
rethinkdns.com/configure is just a webpage. The configuration it produces after being redirected is of the origin.

Also see #169

**Comment by suricloud** [comment]
ok,
when blocklist is selected with selfhosted doh url, does the selected blockist gets updated automatically?
or should we run something on github to get the latest update from the blocklist?

**Reply by ignoramous** [reply]
Blocklists get updated once a week, and pushed out only on code *deploy*s. 

To automate *deploys*, see: https://github.com/serverless-dns/serverless-dns/discussions/149
